src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -104,7 +104,8 @@ class SendCoinsConfirmScreen extends Component<Props> {
       { txDecoded },
       {
         onSuccess: (txid: string) => {
-          const trimmedMemo = memo.trim();
```

Comment:
what does it fix? Was it on purpose changed?

Reply:
![image](https://user-images.githubusercontent.com/67273035/99947417-3eed5a00-2d78-11eb-9d92-84395881be73.png)
it fixes the error of undefined memo when sending transaction signed from the authenticator